packages/console/src/common/ObjectIcon.tsx

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { dhTable, vsGraph, vsPreview } from '@deephaven/icons';
 import dh from '@deephaven/jsapi-shim';
-import { ReactComponent as PandasIcon } from '../../assets/svg/pandas_mark_white_tiny.svg';
```

Comment:
If we don't need the Pandas icon, maybe delete the svg too, to not keep the unused asset in the codebase.

Reply:
Why are we dropping the pandas icon?

Reply:
It should be put into @deephaven/icons instead #37 created

Reply:
Can't, it's multi-colored.

Reply:
This merged with no resolution, and no follow up tickets.

Reply:
It'll be captured in #37 . If we can't add it to @deephaven/icons, then need to add direct to @deephaven/console, and have `svgr` somewhere in the pipeline.